### PR TITLE
[voice][v1] Add sipbreakout v1 proto

### PR DESCRIPTION
**Pull request by noy-wg2**

### CODE REVIEW

File: wgtwo/sipbreakout/v1/sipbreakout.proto [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+//  TLS is used as SIP transport and TAS shall present valid certificate for SIP URI stated in
+//  Registration.sip_uri
+//
+// Media
+//  WebRTC style SDPs are used between WG2 SBC and TAS to encrypt media sessions.
+service SipBreakoutService {
+
+  // Add or replace a registration
+  rpc UpsertRegistration (UpsertRegistrationRequest) returns (UpsertRegistrationResponse) {
+    option (scope) = "call.control:answer_and_initiate";
```
Comment (noy-wg2) [inline wgtwo/sipbreakout/v1/sipbreakout.proto:88]: Should the scope be `call.control:loop_and_fork`?

Comment (noy-wg2) [reply wgtwo/sipbreakout/v1/sipbreakout.proto:88]: `call.routing:control`?

Comment (gi-wg2) [inline wgtwo/sipbreakout/v1/sipbreakout.proto:88]: Just to have a placeholder for the discussion 🙃 
`call.routing:write`?

**Comment by noy-wg2** [comment]
@gi-wg2 which other files I need to change? 
https://github.com/omnicate/loltel/blob/develop/proto/scope/scope.proto this one and?

**Comment by noy-wg2** [comment]
Was done as part of a PR in loltel

**Pull request closed without merging**